cmd/devtool/devtool.go

```diff
@@ -330,13 +330,10 @@ func ethSetup(ethAcctAddr, keystoreDir string, isBroadcaster bool) {
 func createTranscoderRunScript(ethAcctAddr, dataDir, serviceHost string) {
```

Comment:
Not related to your PR, but we could clean up a little bit the generated scripts, now some of the commands are in new lines, some of them are in the same, I'd make it consistent. Wdyt?

Generated `run_broadcaster.sh` (some parameters are in new lines, some are in the same):
```
#!/bin/bash
./livepeer -v 99 -ethController 0x77a0865438f2efd65667362d4a8937537ca7a5ef -datadir ./.lpdev2/broadcaster_0x06AAaf48472D6a64f33C7e281b5dB0fE419878F6 \
    -ethAcctAddr 0x06AAaf48472D6a64f33C7e281b5dB0fE419878F6 \
    -ethUrl http://localhost:8545/ \
    -ethPassword "" \
    -network=devenv \
    -blockPollingInterval 1 \
    -monitor=false -currentManifest=true -cliAddr 127.0.0.1:7935 -httpAddr 127.0.0.1:8935  -broadcaster=true -rtmpAddr 127.0.0.1:1935
```

Generated `run_trascoder.sh` (everything is in one line):
```
#!/bin/bash
./livepeer -v 99 -datadir ./.lpdev2/transcoder_0x065195849B39Bf5D99b4EbD28F61C75C6EB8771a -orchSecret secre -orchAddr 127.0.0.1:8936 -cliAddr 127.0.0.1:7937 -transcoder
```

Reply:
@thomshutt Could you check this as well?